ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -131,7 +131,7 @@ public ConnectSchema valueConnectSchema() {
   }
 
   @VisibleForTesting
-  public Optional<Column> findColumn(final String columnName) {
+  private Optional<Column> findColumn(final String columnName) {
```

Comment:
Do we still need the `@VisibleForTesting` annotation if the method is private?

Reply:
derp! I'll fix this in a follow-up PR so I don't have to re-test (i'm touching this code anyway). Thanks for the review!